### ISSUE TITLE
Fix Fathom integration

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/covid-19-procurement-explorer-public#347]
This repo uses different environment variables than those set in the deploy repo.

Also, we should set `data-spa="auto"` as documented here: https://usefathom.com/integrations/other

**Comment by KushalRaj** [comment]
Please specify what should we set in the `src` and `data-site`. The current variables were set as per you description in issue #275

**Comment by jpmckinney** [comment]
The problem is that in the deploy repo these environment variables are set:

* REACT_APP_FATHOM_ANALYTICS_DOMAIN
* REACT_APP_FATHOM_ANALYTICS_ID

Whereas `FathomAnalytics.js` expects:

* REACT_APP_FATHOM_SRC
* REACT_APP_FATHOM_SITE_KEY

Here's a diff:

```diff
diff --git a/src/FathomAnalytics.js b/src/FathomAnalytics.js
index 01a925e..babd771 100644
--- a/src/FathomAnalytics.js
+++ b/src/FathomAnalytics.js
@@ -1,15 +1,16 @@
 import React from 'react'
 
 const FathomAnalytics = () => {
-    const src = process.env.REACT_APP_FATHOM_SRC
-    const siteKey = process.env.REACT_APP_FATHOM_SITE_KEY
+    const domain = process.env.REACT_APP_FATHOM_ANALYTICS_DOMAIN || 'cdn.usefathom.com'
+    const siteKey = process.env.REACT_APP_FATHOM_ANALYTICS_ID
     const excludedDomains = 'localhost,127.0.0.1,0.0.0.0'
 
     return (
         <script
-            src={src}
+            src="https://{domain}/script.js"
             data-site={siteKey}
             data-excluded-domains={excludedDomains}
+            data-spa="auto"
             defer
         />
     )
```

**Comment by jpmckinney** [comment]
Fixed in #353
